src/dashboards/components/DashboardPage.tsx

```diff
@@ -150,7 +150,7 @@ const mstp = (state: AppState) => {
     state.currentDashboard.id
   )
 
-  const showAnnotationBar = state.userSettings.showAnnotationsControls
+  const showAnnotationBar = state.userSettings.showAnnotationsControls || false
```

Comment:
i think we want to use `??` here instead of `||`:

https://dev.to/hereisnaman/logical-or-vs-nullish-coalescing-operator-in-javascript-3851